.ci/install/isal.sh

```diff
@@ -2,15 +2,15 @@
 
 set -ex
 
-ISAL_MIRROR=http://kevinlin.web.rice.edu/static/isal.tar.gz
+ISAL_MIRROR=http://kevinlin.web.rice.edu/static/isal-2.tar.gz
 
 cd $HOME
-if [ ! -d /tmp/cache/isal ]; then
-    wget --quiet -O isal.tar.gz $ISAL_MIRROR
-    tar -xf isal.tar.gz -C /tmp
-    mv /tmp/isa-l_open_src_2.13 /tmp/cache/isal
-    cd /tmp/cache/isal
-    make
-    cd $HOME
-fi
-ln -s /tmp/cache/isal isal
+wget --quiet -O isal.tar.gz $ISAL_MIRROR
+tar -xf isal.tar.gz
+cd $HOME/isal
+./autogen.sh
+./configure
+make
+sudo make install
+cd $HOME
+ln -s $HOME/isal isal
```

Comment:
Does this work? Seems like it's creating a symlink to itself? I think you can probably delete this line entirely

Reply:
shoot accidentally merged without seeing this. will make a quick oneliner pr for this. my bad.